test/controllers/sections/destination/DestinationIndexControllerSpec.scala

```diff
@@ -60,6 +61,19 @@ class DestinationIndexControllerSpec extends SpecBase with MockUserAnswersServic
       }
     }
 
+    "when DestinationSection is not completed, and DestinationSection.status == UpdateNeeded" - {
```

Comment:
```suggestion
    "when DestinationSection.status == UpdateNeeded" - {
```

Reply:
I wanted to be explicit about this, as the DestinationSection not being completed is the only way the second part of the or condition is evaluated